.github/workflows/lint-pr.yml

```diff
@@ -0,0 +1,38 @@
+name: Lint PR
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
```

Comment:
Add `synchronize`? 
```suggestion
      - edited
      - synchronize
```

Reply:
Yes, I have a done now. Had to test on a branch inside the repo.